shared/constants/gas.ts

```diff
@@ -3,6 +3,7 @@ import { addHexPrefix } from 'ethereumjs-util';
 const ONE_HUNDRED_THOUSAND = 100000;
 const MIN_GAS_LIMIT_DEC = '21000';
 
+export const TOO_HIGH_GAS_LIMIT = '1'; // 1 ETH
```

Comment:
[suggestion]

rather than add the comment, we can include it in the variable name 
```suggestion
export const GAS_LIMIT_TOO_HIGH_IN_ETH = '1';
```
- too high suggests 1 ETH will trigger the warning, but the warning appears after 1 ETH

--- 

have we considered including the variable directly in the file? I wonder what the likelihood is for us reusing this

Reply:
Yeah not sure if we are going to re-use as well.